packages/block-editor/src/components/inserter-listbox/item.js

```diff
@@ -33,11 +33,7 @@ function InserterListboxItem(
 					return children( propsWithTabIndex );
 				}
 				return (
-					<Button
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
-						{ ...propsWithTabIndex }
-					>
+					<Button __next40pxDefaultSize { ...propsWithTabIndex }>
```

Comment:
The custom style height: auto is causing it to be `90px`, so `__next40pxDefaultSize` has no effect in this case. I have not changed the styles as it was looking good. Let me know if anything needs to be changed.

![inserter-lightbox-button-before](https://github.com/user-attachments/assets/5a995764-5ab8-4d90-9024-e80693886174)